ModKit/UI/CharCodeGlyphs.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+namespace ModKit {
+    public static partial class Glyphs {
+        public const string CharCodeCheckOn = "x";
+        public const string CharCodeCheckOff = "o";
+        public const string CharCodeCheckEmpty = "[]";
+        public const string CharCodeDisclosureOn = "v";
+        public const string CharCodeDisclosureOff = ">";
+        public const string CharCodeDisclosureEmpty = "[]";
```

Comment:
For disclosure empty please use a space and consider using + and - for v and >

Reply:
Done!